include/IECore/MeshAlgo.h

```diff
@@ -55,6 +55,7 @@ std::pair<PrimitiveVariable, PrimitiveVariable> calculateTangents( const MeshPri
 );
 
 void resamplePrimitiveVariable( const MeshPrimitive *mesh, PrimitiveVariable& primitiveVariable, PrimitiveVariable::Interpolation interpolation );
+MeshPrimitivePtr deleteFaces( const MeshPrimitive *meshPrimitive, const PrimitiveVariable &primvar );
```

Comment:
Lets add a docstring clarifying that the primvar is expected to be a Uniform mask. Should we also name the arg something more meaningful? "mask" or "facesToKeep" or ... ?